README.md

```diff
@@ -1,28 +1,78 @@
-# CMake Tools
+<h1 align=center> <img align=center width=48 src="res/logo-margin-bottom.png" alt="CMake Logo"/> CMake Tools </h1>
 
+<p align=center ><i>Enhancing CMake development in VS Code.</i></p>
+
+<div align=center>
+    <img alt="VS Code" src="https://img.shields.io/badge/VSCode-7d57c2?style=for-the-badge&logo=visual%20studio%20code&logoColor=white"/>
+    <img alt="CMake" src="https://img.shields.io/badge/CMake-7d57c2?style=for-the-badge&logo=cmake&logoColor=white"/>
+    <img alt="C" src="https://img.shields.io/badge/C-7d57c2?style=for-the-badge&logo=c&logoColor=white"/>
+    <img alt="C++" src="https://img.shields.io/badge/C%2B%2B-7d57c2?style=for-the-badge&logo=c%2B%2B&logoColor=white"/>
+</div>
+
+<div align=center>
+    <img alt="VS Code Marketplace Downloads" src="https://img.shields.io/visual-studio-marketplace/d/ms-vscode.cmake-tools?style=for-the-badge&color=064f8c&cacheSeconds=3600&link=https%3A%2F%2Fmarketplace.visualstudio.com%2Fitems%3FitemName%3Dms-vscode.cmake-tools">
+    <img alt="VS Code Marketplace Version" src="https://img.shields.io/visual-studio-marketplace/v/ms-vscode.cmake-tools?style=for-the-badge&color=be2128&cacheSeconds=3600&link=https%3A%2F%2Fmarketplace.visualstudio.com%2Fitems%2Fms-vscode.cmake-tools%2Fchangelog">
+    <img alt="Closed Issues" src="https://img.shields.io/github/issues-closed/microsoft/vscode-cmake-tools?style=for-the-badge&color=249847&cacheSeconds=3600&link=https%3A%2F%2Fgithub.com%2Fmicrosoft%2Fvscode-cmake-tools%2Fissues">
+</div>
+
+<br/>
+<img style="width: 100%" src="res/graphics/cmake-tools-banner.png" alt="CMake Tools Banner"/>
+
+## Introduction
 [CMake Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cmake-tools) provides the native developer a full-featured, convenient, and powerful workflow for CMake-based projects in Visual Studio Code.
 
-## Important doc links
+<details open>
+<summary>
+ Features
+</summary><br/>
+
+<p align=center>
+    <img width="49%" src="res/graphics/presets-support-feature.png" alt="Presets Support"/>
+&nbsp;
+    <img width="49%" src="res/graphics/cmake-debugging-feature.png" alt="CMake Debugging"/>
+</p>
+<p align=center>
+    <img width="49%" src="res/graphics/cmake-language-service-feature.png" alt="Built-In CMake Language Services"/>
+&nbsp;
+    <img width="49%" src="res/graphics/test-integration-feature.png" alt="Test Integration"/>
+</p>
+<p align=center>
+    <img width="49%" src="res/graphics/cmake-cache-editor-feature.png" alt="CMake Cache Editor"/>
+&nbsp;
+    <img width="49%" src="res/graphics/status-and-sidebar-feature.png" alt="Status & Sidebar"/>
+</p>
+<p align=center>
+    <img width="49%" src="res/graphics/cmake-commands-feature.png" alt="CMake Commands"/>
+&nbsp;
+    <img width="49%" src="res/graphics/tasks-feature.png" alt="Tasks"/>  
+</p>  
+
+</details>
+
+## Setup and Installation
+[Download CMake](https://cmake.org/download/) and add it to PATH.
```

Comment:
Maybe let's be more nuanced with this. They could get CMake from a VS installation and edit the cmake.cmakePath, or they could install it directly, maybe we could mention both.

Reply:
Updated this.